daliuge-engine/dlg/data/drops/memory.py

```diff
@@ -61,11 +61,12 @@ def parse_pydata(pd: Union[bytes, dict]) -> bytes:
     builtin_types = get_builtins()
     if pd_dict["type"] != "raw" and type(pydata) in builtin_types.values() and pd_dict["type"] not in builtin_types.keys():
         logger.warning("Type of pydata %s provided differs from specified type: %s", type(pydata).__name__, pd_dict["type"])
-        pd_dict["type"] = type(pydata).__name__
+        # pd_dict["type"] = type(pydata).__name__
```

Comment:
We have too much commented-out code in DALiuGE and I am reticent to keep this here if it is not necessary.

 What was the actual error this was causing?